rebasehelper/logger.py

```diff
@@ -30,47 +32,79 @@ class LoggerHelper(object):
 
     @staticmethod
     def get_basic_logger(logger_name, level=logging.DEBUG):
-        """
-        Sets-up a basic logger without any handler
+        """Sets up a basic logger without any handler.
+
+        Args:
+            logger_name (str): Logger name.
+            level (int): Severity level set to the logger.
```

Comment:
`Severity level set to the logger` - that doesn't make much sense. How about `Severity level associated with the logger`?